src/spec-lang/ast/number.ts

```diff
@@ -1,14 +1,35 @@
 import { SNode, Range } from "./node";
 import { BigInteger } from "big-integer";
+import bigInt from "big-integer";
+
+const units = new Map<string, BigInteger>([
+    ["wei", bigInt(1)],
+    ["gwei", bigInt(1e9)],
+    ["ether", bigInt(1e18)],
+    ["seconds", bigInt(1)],
+    ["minutes", bigInt(60)],
+    ["hours", bigInt(3600)],
+    ["days", bigInt(86400)],
+    ["weeks", bigInt(604800)]
+]);
 
 export class SNumber extends SNode {
     public readonly num: BigInteger;
     public readonly radix: number;
+    private readonly conversionMultipler: Map<string, BigInteger> = units;
```

Comment:
This private variable is unnecessary. Should just rename the file-level `units` constant to `unitMultiplier` and use that instead.

Reply:
Fixed in 4f5f56a